custom_components/ha-google-home/api.py

```diff
@@ -104,10 +116,26 @@ def get_google_devices_information(self):
                             device=device.device_name, error=API_RETURNED_UNKNOWN
                         )
                     )
+            else:
+                offline_devices.append(device.device_name)
```

Comment:
I think we should add the ip check inside of the `for` loop 
Sort of 
```
for device in devices:
...
if device.ip:
  timers = []
  alarms = []
  result = ...
else:
  offline_devices.append(...)
```

Reply:
I will moved it